reana_client/cli/__init__.py

```diff
@@ -29,7 +29,55 @@ def __init__(self):
         self.reana_server_url = os.getenv('REANA_SERVER_URL', None)
 
 
-@click.group()
+class ReanaCLI(click.Group):
+    """Reana command line interface."""
+
+    cmd_groups = [ping.configuration_group,
```

Comment:
Looking at https://github.com/reanahub/reana-client/issues/252, I find `Workspace interactive commands` and `Workflow management commands` groups missing.